src/fsharp/FSharp.Core/map.fs

```diff
@@ -729,6 +769,10 @@ module Map =
     let add key value (table: Map<_, _>) =
         table.Add (key, value)
 
+    [<CompiledName("Change")>]
```

Comment:
This is a new FSharp.Core api, and so should have the ExperimentalAttribute applied, whilst in Preview.

Example:
https://github.com/dotnet/fsharp/blob/ed9b9b7df908bbc2b90e46de27dc05678a6d787a/src/fsharp/FSharp.Core/quotations.fsi#L108